### PR TITLE
Announcing Scala.js 0.6.11.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

**Comment by sjrd** [comment]
Review by @nicolasstucki and @gzm0 
Planned release date: Wednesday July 27.

**Comment by nicolasstucki** [comment]
LGTM

File: _posts/news/2016-07-27-announcing-scalajs-0.6.11.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+## Potentially breaking change
+
+This release fixes [#2445](https://github.com/scala-js/scala-js/issues/2445), which potentially constitutes a breaking change, if you relied on the previously buggy behavior.
+Recall that, in JavaScript types and exports, `def`s without parentheses are JavaScript *getters*.
+For example,
+
+{% highlight scala %}
+@js.native
+class Foo {
```
Comment (gzm0) [inline _posts/news/2016-07-27-announcing-scalajs-0.6.11.md]: extends js.Object

**Comment by gzm0** [comment]
LGTM otherwise

File: _posts/news/2016-07-27-announcing-scalajs-0.6.11.md (hunk 1 above)
Comment (sjrd) [inline _posts/news/2016-07-27-announcing-scalajs-0.6.11.md]: Good catch. Fixed.

File: _posts/news/2016-07-27-announcing-scalajs-0.6.11.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+val foo = new Foo
+val a = foo.makeArray // in JavaScript, this is foo.makeArray, *not* foo.makeArray()
+{% endhighlight %}
+
+The bug [#2445](https://github.com/scala-js/scala-js/issues/2445) was that, if `makeArray` was defined with type parameters, it would erroneously translate to a *method call* in JavaScript, rather than a *field access/getter call*.
+
+{% highlight scala %}
+@js.native
+class Foo {
```
Comment (gzm0) [inline _posts/news/2016-07-27-announcing-scalajs-0.6.11.md]: Ah, here as well :P

Comment (sjrd) [inline _posts/news/2016-07-27-announcing-scalajs-0.6.11.md]: Duh ^^

**Comment by sjrd** [comment]
Artifacts are available on Maven Central; APIs and CLIs are up on the website.

We're all set for announcement tomorrow morning :-)

**Comment by gzm0** [comment]
LGTM

**Pull request merged**